server/src/main/scala/za/co/absa/atum/server/api/http/Endpoints.scala

```diff
@@ -75,6 +77,15 @@ trait Endpoints extends BaseEndpoints {
       .out(jsonBody[SingleSuccessResponse[AdditionalDataSubmitDTO]])
   }
 
+  protected val getPartitioningCheckpointEndpointV2
+    : PublicEndpoint[(Long, UUID), ErrorResponse, SingleSuccessResponse[CheckpointV2DTO], Any] = {
```

Comment:
GetCheckpointV2Args can be used here

Reply:
I can't be done that way. It has to be a tuple in this form. Feel free to experiment ....